### ISSUE TITLE
qt5-webkit not building

### ISSUE THREAD

**Issue by steev182** [adlerweb/davinci-resolve-studio-beta#2]
qt5-webkit no longer builds, so this would prevent anyone from installing resolve after fixing their updates. 

I've just tried removing it from the dependencies with your pkgbuild, and Resolve seems to install and start. Just waiting on it to optimize neural engines and I'll be able to confirm if it really is ok now.

**Comment by adlerweb** [comment]
Yepp, the package is no longer actively maintained by its author. Try [qt5-webkit-movableink-git](https://aur.archlinux.org/packages/qt5-webkit-movableink-git) instead, it should provide qt5-webkit and should still work.

**Comment by steev182** [comment]
Yeah, qt5-webkit-moveableink-git wasn't building for me either. However, I'm beginning to think qt5-webkit is an old dependency as my first project in 19 without listing it as a dependency or having it installed worked flawlessly.
